docker-compose.dev.yml

```diff
@@ -1,33 +1,17 @@
+## **************** DOCUMENTATION ****************
+##
+## Please refer to the documentation for more information on how to use this file. 
+## https://www.activepieces.com/docs/contributing/development-setup/docker-compose
+##
+## ***********************************************
 version: '3.0'
 
 services:
-  app:
```

Comment:
I would strongly suggest to keep it and use [profiles](https://docs.docker.com/compose/profiles/) instead

Reply:
Done!